test/verify/check-static-login

```diff
@@ -740,6 +740,9 @@ matchrule = <SUBJECT>^DC=LAN,DC=COCKPIT,CN=alice$
         # it into the browser.
 
         def do_test(authopts, expected, not_expected=[], session_leader=None):
+            # clear sssd logs; this makes debugging individual failures easier, and also works around https://bugs.debian.org/1001377
+            m.execute("rm -f /var/log/sssd/*")
```

Comment:
Let's only do this on debian-testing; the bug is distro specific for now, and I want to at least know when it starts to happen in Fedora as well.

Reply:
Done.